constraints.pro

```diff
@@ -204,8 +204,9 @@ gen_enforced_field(WorkspaceCwd, 'scripts.build', 'tsup --clean && yarn build:ty
   \+ workspace_field(WorkspaceCwd, 'private', true),
   WorkspaceCwd \= '.',
   WorkspaceCwd \= 'packages/snaps-simulator',
-  WorkspaceCwd \= 'packages/snaps-cli'.
-gen_enforced_field(WorkspaceCwd, 'build:types', 'tsc --project tsconfig.build.json') :-
+  WorkspaceCwd \= 'packages/snaps-cli',
+  WorkspaceCwd \= 'packages/snaps-execution-environments'.
```

Comment:
Why are we making this change?

Reply:
I changed the build command in `snaps-execution-environments` to build the LavaMoat packages too.

Reply:
Good thing Yarn 4 supports JavaScript-based constraints. 😅